### ISSUE TITLE
Feature of sharing about sir in WhatsApp so that other can download the App

### ISSUE THREAD

**Issue by soumyajit1729** [soumyajit1729/Smart-Teaching#3]
Make the facility of sharing some content from our app to any person in WhatsApp/Messenger

**Comment by pawank0411** [comment]
@soumyajit1729 can you please explain this issue

**Comment by soumyajit1729** [comment]
yes of course, so often you will see in various apps that share this news with 5 people in WhatsApp and you will earn this that. This the feature I am talking about. Sharing some news through our app.
